tests/client/test_client_key_meilisearch.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from tests import common
+from datetime import datetime, timedelta
+
+def test_get_keys_default(client):
+    """Tests if public and private keys have been generated and can be retrieved."""
+    key = client.get_keys()
+    assert isinstance(key, list)
+    assert len(key) >= 2
+    assert 'actions' in key[0]
+    assert 'indexes' in key[0]
+    assert key[0]['key'] is not None
+    assert key[1]['key'] is not None
+
+def test_get_key(client):
+    """Tests if a key can be retrieved."""
+    keys = client.get_keys()
+    key = client.get_key(keys[0]['key'])
+    assert isinstance(key, dict)
+    assert 'actions' in key
+    assert 'indexes' in key
+    assert key['createdAt'] is not None
+
+def test_get_key_inexistent(client):
+    """Tests getting a key that does not exists."""
+    with pytest.raises(Exception):
+        client.get_key('No existing key')
+
+def test_create_keys_default(client):
```

Comment:
A general suggestion for all the key tests that create/update/delete a key is to reset or remove the key after the test runs. Otherwise tests can become dependent on the order they are run. The only test I currently see that would be dependent on order is `test_get_keys_default` so maybe not a huge deal right now, but in the future new tests could be added where it matters and it may not be immediately apparent why they are failing.

Starting every test with only the 2 unchanged default keys will prevent tests for affecting each other and causing unexpected outcomes.